src/Microsoft.DotNet.XHarness.TestRunners.Xunit/ThreadlessXunitTestRunner.cs

```diff
@@ -93,9 +93,6 @@ public override async Task Run(IEnumerable<TestAssemblyInfo> testAssemblies)
             _assembliesElement.Add(resultsXmlAssembly);
         }
         TotalTests = totalSummary.Total;
-        Console.WriteLine($"{Environment.NewLine}=== TEST EXECUTION SUMMARY ==={Environment.NewLine}Total: {totalSummary.Total}, Errors: 0, Failed: {totalSummary.Failed}, Skipped: {totalSummary.Skipped}, Time: {TimeSpan.FromSeconds((double)totalSummary.Time).TotalSeconds}s{Environment.NewLine}");
```

Comment:
I'd keep this, it's quite nice to have a summary at the end when looking at console logs.

Reply:
There is summary implemented in the inherited class. I have removed a duplicate, though the one from the threadless was more eye catching 🤷‍♂️

```
info: Tests run: 2 Passed: 2 Inconclusive: 0 Failed: 0 Ignored: 0 Skipped: 0
```

Reply:
Add the `=== TEST EXECUTION SUMMARY ===` to the common code ?